GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
Ugh.

```
[17:41:22][Exec] git config --global user.name "git-tfs"
[17:41:22][Exec] error: could not lock config file (null)/(null)/.gitconfig: No such file or directory
```

It looks like CI.proj should set the HOMEDRIVE and HOMEPATH env vars? found in http://chrisoldwood.blogspot.com/2013/12/missing-homedrive-and-homepath.html